### ISSUE TITLE
Implement a tuorial on first start for the menus, explaining general stuff about game

### ISSUE THREAD

**Issue by def-** [ddnet/ddnet#2583]
Best would be a tut that does this dim thingy and shows what's important, and then some button that shows it all again, the thing is, if they go super fast through the tut, they still passively internalize the "flow" of ui navigation, which is way better than having to read a bunch and find yourself out whats meant @deen.

As suggested by Khejanin on Discord.

**Comment by heinrich5991** [comment]
Duplicate of #269.

**Comment by def-** [comment]
No, this one is about the menus etc only, not about teaching gameplay. But strongly related, yeah.

**Comment by heinrich5991** [comment]
Fixed by #4876.

**Comment by C0D3D3V** [comment]
I do not think it is fixed. Since this is still about the menu and not about gameplay.
